common/protob/messages-solana.proto

```
@@ -38,5 +53,39 @@
+    optional bytes application_domain = 5;  // application domain must be 32 bytes
+}
+
+/**
+ * Request: ask device to sign arbitrary message except valid solana transaction
+ * @start
+ * @next SolanaMessageSignature
+ * @next Failure
+ */
+message SolanaSignUnsafeMessage {
+    repeated uint32 address_n = 1;          // BIP-32 path to derive the key from master node
+    required bytes message = 2;             // the message to sign
```

Buf (1.47.2)

Field named %!q(MISSING) should not be required.

(FIELD_NOT_REQUIRED)